diff --git a/node_modules/google-closure-library/closure/goog/labs/net/webchannel.js b/node_modules/google-closure-library/closure/goog/labs/net/webchannel.js
index c5b853e..94e343c 100644
--- a/node_modules/google-closure-library/closure/goog/labs/net/webchannel.js
+++ b/node_modules/google-closure-library/closure/goog/labs/net/webchannel.js
@@ -64,7 +64,6 @@ goog.require('goog.net.XmlHttpFactory');
  * to be enabled.
  *
  * @interface
- * @extends {EventTarget}
  * @extends {goog.events.Listenable}
  */
 goog.net.WebChannel = function() {};
diff --git a/node_modules/google-closure-library/closure/goog/testing/fs/file.js b/node_modules/google-closure-library/closure/goog/testing/fs/file.js
index 334ef36..d1706da 100644
--- a/node_modules/google-closure-library/closure/goog/testing/fs/file.js
+++ b/node_modules/google-closure-library/closure/goog/testing/fs/file.js
@@ -50,5 +50,11 @@ goog.testing.fs.File = function(name, opt_lastModified, opt_data, opt_type) {
    * @type {Date}
    */
   this.lastModifiedDate = opt_lastModified || null;
+
+  /**
+   * Don't use this prop. This is only to avoid type error in TypeScript with clutz.
+   * @type {number}
+   */
+  this.lastModified = opt_lastModified && opt_lastModified.getTime() || 0;
 };
 goog.inherits(goog.testing.fs.File, goog.testing.fs.Blob);
diff --git a/node_modules/google-closure-library/closure/goog/ui/colormenubuttonrenderer.js b/node_modules/google-closure-library/closure/goog/ui/colormenubuttonrenderer.js
index 772b3ad..ad200b2 100644
--- a/node_modules/google-closure-library/closure/goog/ui/colormenubuttonrenderer.js
+++ b/node_modules/google-closure-library/closure/goog/ui/colormenubuttonrenderer.js
@@ -79,10 +79,11 @@ goog.ui.ColorMenuButtonRenderer.prototype.createCaption = function(
 /**
  * Wrap a caption in a div with the color-menu-button-indicator CSS class.
  * @param {goog.ui.ControlContent} content Text caption or DOM structure.
- * @param {goog.dom.DomHelper} dom DOM helper, used for document interaction.
+ * @param {goog.dom.DomHelper|string} dom DOM helper, used for document interaction.
+ * @param {goog.dom.DomHelper} dummy Dummy param for compatibility with TypeScript.
  * @return {!Element} Caption element.
  */
-goog.ui.ColorMenuButtonRenderer.wrapCaption = function(content, dom) {
+goog.ui.ColorMenuButtonRenderer.wrapCaption = function(content, dom, dummy) {
   return dom.createDom(
       goog.dom.TagName.DIV,
       goog.getCssName(goog.ui.ColorMenuButtonRenderer.CSS_CLASS, 'indicator'),
